redaxo/src/core/lang/en_gb.lang

```diff
@@ -224,6 +224,8 @@ sql_unable_to_create_database = Unable to create the database!
 sql_unable_to_find_database = Unable to find the defined database!
 sql_unable_to_open_database = Unable to open the definied database!
 
+update_required_install_addon_version = This update requires at least version <b>{0}</b> of the <b>install</b> addon!
```

Comment:
Mir fällt gerade auf, während des Updates wird noch nicht die neue Sprachdatei gelesen sein, daher nehme ich die Meldung direkt in die update.php rein (dann nur Englisch).l